test/padCharsEnd.js

```diff
@@ -0,0 +1,185 @@
+import * as RA from '../src';
+import {
+  padCharsEndStrInvoker,
+  padCharsEndStrPolyfill,
+} from '../src/padCharsEnd';
+import eq from './shared/eq';
+
+describe('padCharsEnd', function() {
```

Comment:
Can we add tests that demonstrates what happens when `targetLength` is a negative number ?